src/manage_migrators.c

```diff
@@ -1737,6 +1737,60 @@ migrate_208_to_209 ()
   return 0;
 }
 
+/**
+ * @brief Migrate the database from version 208 to version 209.
+ *
+ * @return 0 success, -1 error.
+ */
+int
+migrate_209_to_210 ()
+{
+  sql_begin_immediate ();
+
+  /* Ensure that the database is currently version 209. */
+
+  if (manage_db_version () != 209)
+    {
+      sql_rollback ();
+      return -1;
+    }
+
+  /* Update the database. */
+
+  /* Remove the fields "bid" and "xref" from table "nvts".
+   * This is done by dropping the entire table and
+   * unset the sync time stamps to force gvmd to refresh the content. */
+
+  sql ("DROP TABLE IF EXISTS nvts CASCADE;");
```

Comment:
Wouldn't an ALTER TABLE be simpler?

Reply:
Since it the data can be rebuild easily I thought a clean drop fresh rebuild
is a safe way. It seemed simpler to me.

Reply:
A potential problem is that if a subsequent migrator wants to modify
the nvts table then it won't be there anymore.

I think we can skip the complex SQLite case because there's
no harm in having the extra columns.  We will drop SQLite
soon anyway.  So it should just be two of: ALTER TABLE nvts
DROP COLUMN xxx.

Also the ALTER TABLE matches what will be needed in
gvm-migrate-to-postgres.

Reply:
It is now using ALTER TABLE.